crates/storage/src/connection/trie.rs

```diff
@@ -594,7 +614,7 @@ impl Transaction<'_> {
             .prepare_cached(&format!("SELECT hash FROM {table} WHERE idx = ?"))
             .context("Creating get statement")?;
 
-        stmt.query_row(params![&index], |row| row.get_felt(0))
+        stmt.query_row(params![&index.get()], |row| row.get_felt(0))
```

Comment:
Implementing `pathfinder_storage::params::ToSql` would make it possible to use a `TrieStorageIndex` as a parameter directly. Please check the `to_sql_int!()` macro to implement `ToSql`.